README.md

```diff
@@ -217,6 +217,8 @@ DYNDNS:
 
 Google:
 
+Note that you must enable Dynamic DNS in the Synthetic records section of DNS management. The username and password is generated once you . You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.

```

Comment:
There is a missing piece after `is generated once you`.

Reply:
The text should read:

Note that you must enable Dynamic DNS in the Synthetic records section of DNS management. The username and password is generated once you create the Dynamic DNS entry . You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.